### PR TITLE
feat: settings page enhancements mock

### PR DESCRIPTION
**Pull request by SooditK**
## Description

This PR will implement a design for the `/settings`, `/settings/notifications` and `/settings/appearance` pages. This is currently just a first draft to get some feedback on ideas that could be added:

image

image

image

Extras:

- I also added `radio-group` and `switch` from `shacnui`

fixes: #533

### CODE REVIEW

File: apps/web/src/components/settings/notifications.tsx [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+                      disabled
+                      aria-readonly
+                    />
+                  </FormControl>
+                </FormItem>
+              )}
+            />
+          </div>
+        </div>
+        <FormField
```
Comment (Hacksore) [inline apps/web/src/components/settings/notifications.tsx:253]: We can omit this as there wont be a mobile app at this time

File: apps/web/src/app/settings/page.tsx [hunk 2]
```diff
@@ -36,17 +33,8 @@ export default async function SettingsPage() {
...
-      }}
-      username={profileData.name}
-    />
+    <>
+      <NewSettings user={profileData} />
+    </>
   );
 }
-
-export const metadata: Metadata = {
```
Comment (Hacksore) [inline apps/web/src/app/settings/page.tsx:49]: Can we keep the meta here?

File: apps/web/src/components/settings/new-settings.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            />
+            <UserHeader user={user} isOwnProfile />
+            <p
+              className="text-sm italic tracking-tight text-neutral-500"
+              title={`Joined ${user.createdAt.toString()}`}
+            >
+              Joined {getRelativeTime(user.createdAt)}
+            </p>
+            <div className="mb-4 mt-2 w-full text-center text-sm md:w-64 md:text-start">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Minima temporibus maiores
```
Comment (Hacksore) [inline apps/web/src/components/settings/new-settings.tsx:60]: Since a diff PR updated this can you make this the bio value?

**Review by Hacksore — DISMISSED** [review]
Lost of duplicated code in appearance/notification files but that might be because this is wip.

Just leaving my feedback early to help get this more polished.

**Comment by falcon-head** [comment]
Is it still being worked on?

**Comment by bautistaaa** [comment]
can you check the quality checks, gonna merge this

**Comment by SooditK** [comment]
@bautistaaa sure, i'll update them by lunch

**Comment by SooditK** [comment]
Hii @bautistaaa 
I checked, it's fine

image

**Comment by falcon-head** [comment]
> Hii @bautistaaa 
> 
> I checked, it's fine
> 
> 
> 
> image

Hey it's `checks:fix`

**Comment by bautistaaa** [comment]
ill be taking this over and getting it mergeable this weekend

**Comment by bautistaaa** [comment]
we'll need to add the rich text editor and url sections from @Hacksore original settings page back in but at least this is merged and i can unravel all the pieces later

**Review by bautistaaa — APPROVED** [review]

**Pull request merged**